jsx/Form.js

```diff
@@ -917,6 +918,7 @@ DateElement.defaultProps = {
   label: '',
   value: '',
   id: null,
+  maxYear: '9999-12-31',
```

Comment:
should we not define `min` as well as `1000-01-01` ??

Reply:
yes, probably! good catch. you are currently able to save year i.e. 0111

Reply:
@ridz1208 i made min year 1900, please re-review

Reply:
why not 1000 ?

Reply:
i dont know, because i based it on the decisions made in https://github.com/aces/Loris/pull/3769, which i guess got changed to 1000 anyways

Reply:
mhmm